crypto/multikey/kind_test.go

```diff
@@ -17,14 +17,16 @@ package multikey
 import (
 	"testing"
 
+	"github.com/mailchain/mailchain/crypto/sr25519/sr25519test"
+
 	"github.com/mailchain/mailchain/crypto"
 	"github.com/mailchain/mailchain/crypto/ed25519/ed25519test"
 	"github.com/mailchain/mailchain/crypto/secp256k1/secp256k1test"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestKeyKindFromSignature(t *testing.T) {
-	assert := assert.New(t)
+	assertion := assert.New(t)
```

Comment:
I have come to realise that `assert := assert.New(t)` is not a good approach. For two reasons I can see now, the first is what you have addressed; changing what `assert` does part the want through a function. Second is that it gives less information about the test case that failed. Can you remove this line and instead add `t` to the assert calls.

Reply:
Sure